diagnostic.md

```diff
@@ -69,13 +66,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 ```
 
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
-
- <!-- Answer Starts Here -->
+relative
+ <!-- Answer Starrelativets Here -->
 
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
-
+relative
```

Comment:
Apologies, I was running out of time so I felt like answering all of them quickly was better than not at all. Thanks!